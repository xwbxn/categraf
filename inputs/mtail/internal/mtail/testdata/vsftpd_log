Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "230 Login successful."
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP command: Client "192.0.2.141", "STAT"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "211-FTP server status:"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     Connected to "
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "192.0.2.141"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     Logged in as "
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "ftp"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     TYPE: "
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "ASCII??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     No session bandwidth limit??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     Session timeout in seconds is "
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "300"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     Control connection is plain text??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     Data connections will be plain text??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     At session startup, client count was "
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "1"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     vsFTPd 2.2.2 - secure, fast, stable??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "211 End of status"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP command: Client "192.0.2.141", "PASV"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "227 Entering Passive Mode (192,0,2,208,85,128)."
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP command: Client "192.0.2.141", "STAT"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "211-FTP server status:"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     Connected to "
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "192.0.2.141"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     Logged in as "
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "ftp"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     TYPE: "
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "ASCII??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     No session bandwidth limit??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     Session timeout in seconds is "
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "300"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     Control connection is plain text??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     Data connections will be plain text??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     At session startup, client count was "
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "1"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "     vsFTPd 2.2.2 - secure, fast, stable??"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "211 End of status"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP command: Client "192.0.2.141", "QUIT"
Mon Feb 21 17:44:19 2011 [pid 3] [ftp] FTP response: Client "192.0.2.141", "221 Goodbye."
Mon Feb 21 17:44:30 2011 [pid 2] CONNECT: Client "192.0.2.143"
Mon Feb 21 17:44:30 2011 [pid 2] FTP response: Client "192.0.2.143", "220 (vsFTPd 2.2.2)"
Mon Feb 21 17:44:31 2011 [pid 2] FTP command: Client "192.0.2.143", "USER anonymous"
Mon Feb 21 17:44:31 2011 [pid 2] [anonymous] FTP response: Client "192.0.2.143", "331 Please specify the password."
Mon Feb 21 17:44:31 2011 [pid 2] [anonymous] FTP command: Client "192.0.2.143", "PASS <password>"
Mon Feb 21 17:44:31 2011 [pid 1] [ftp] OK LOGIN: Client "192.0.2.143", anon password "?"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "230 Login successful."
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP command: Client "192.0.2.143", "STAT"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "211-FTP server status:"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     Connected to "
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "192.0.2.143"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     Logged in as "
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "ftp"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     TYPE: "
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "ASCII??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     No session bandwidth limit??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     Session timeout in seconds is "
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "300"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     Control connection is plain text??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     Data connections will be plain text??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     At session startup, client count was "
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "1"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     vsFTPd 2.2.2 - secure, fast, stable??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "211 End of status"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP command: Client "192.0.2.143", "PASV"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "227 Entering Passive Mode (192,0,2,208,141,0)."
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP command: Client "192.0.2.143", "STAT"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "211-FTP server status:"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     Connected to "
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "192.0.2.143"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     Logged in as "
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "ftp"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     TYPE: "
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "ASCII??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     No session bandwidth limit??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     Session timeout in seconds is "
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "300"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     Control connection is plain text??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     Data connections will be plain text??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     At session startup, client count was "
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "1"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "     vsFTPd 2.2.2 - secure, fast, stable??"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "211 End of status"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP command: Client "192.0.2.143", "QUIT"
Mon Feb 21 17:44:32 2011 [pid 3] [ftp] FTP response: Client "192.0.2.143", "221 Goodbye."
Mon Feb 21 18:31:52 2011 [pid 2] CONNECT: Client "198.51.100.36"
Mon Feb 21 18:31:52 2011 [pid 2] FTP response: Client "198.51.100.36", "220 (vsFTPd 2.2.2)"
Mon Feb 21 18:31:52 2011 [pid 2] FTP command: Client "198.51.100.36", "USER anonymous"
Mon Feb 21 18:31:52 2011 [pid 2] [anonymous] FTP response: Client "198.51.100.36", "331 Please specify the password."
Mon Feb 21 18:31:52 2011 [pid 2] [anonymous] FTP command: Client "198.51.100.36", "PASS <password>"
Mon Feb 21 18:31:52 2011 [pid 1] [ftp] OK LOGIN: Client "198.51.100.36", anon password "anonymous@"
Mon Feb 21 18:31:53 2011 [pid 3] [ftp] FTP response: Client "198.51.100.36", "230 Login successful."
Mon Feb 21 18:31:53 2011 [pid 3] [ftp] FTP command: Client "198.51.100.36", "TYPE I"
Mon Feb 21 18:31:53 2011 [pid 3] [ftp] FTP response: Client "198.51.100.36", "200 Switching to Binary mode."
Mon Feb 21 18:31:54 2011 [pid 3] [ftp] FTP command: Client "198.51.100.36", "PASV"
Mon Feb 21 18:31:54 2011 [pid 3] [ftp] FTP response: Client "198.51.100.36", "227 Entering Passive Mode (192,0,2,208,180,139)."
Mon Feb 21 18:31:54 2011 [pid 3] [ftp] FTP command: Client "198.51.100.36", "STOR /incoming/example_324592677_all.deb"
Mon Feb 21 18:31:54 2011 [pid 3] [ftp] FTP response: Client "198.51.100.36", "150 Ok to send data."
Mon Feb 21 18:31:54 2011 [pid 3] [ftp] OK UPLOAD: Client "198.51.100.36", "/incoming/example_324592677_all.deb", 530 bytes, 1.56Kbyte/sec
Mon Feb 21 18:31:54 2011 [pid 3] [ftp] FTP response: Client "198.51.100.36", "226 Transfer complete."
Mon Feb 21 18:31:54 2011 [pid 3] [ftp] FTP command: Client "198.51.100.36", "TYPE I"
Mon Feb 21 18:31:54 2011 [pid 3] [ftp] FTP response: Client "198.51.100.36", "200 Switching to Binary mode."
Mon Feb 21 18:31:55 2011 [pid 3] [ftp] FTP command: Client "198.51.100.36", "PASV"
Mon Feb 21 18:31:55 2011 [pid 3] [ftp] FTP response: Client "198.51.100.36", "227 Entering Passive Mode (192,0,2,208,182,218)."
Mon Feb 21 18:31:55 2011 [pid 3] [ftp] FTP command: Client "198.51.100.36", "STOR /incoming/example_324592677_all.changes"
Mon Feb 21 18:31:55 2011 [pid 3] [ftp] FTP response: Client "198.51.100.36", "150 Ok to send data."
Mon Feb 21 18:31:55 2011 [pid 3] [ftp] OK UPLOAD: Client "198.51.100.36", "/incoming/example_324592677_all.changes", 752 bytes, 2.25Kbyte/sec
Mon Feb 21 18:31:55 2011 [pid 3] [ftp] FTP response: Client "198.51.100.36", "226 Transfer complete."
Mon Feb 21 18:31:55 2011 [pid 3] [ftp] FTP command: Client "198.51.100.36", "QUIT"
Mon Feb 21 18:31:55 2011 [pid 3] [ftp] FTP response: Client "198.51.100.36", "221 Goodbye."
