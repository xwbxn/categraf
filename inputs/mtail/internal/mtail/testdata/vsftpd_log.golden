counter transfers {direction=}
counter transfer_time {direction=}
counter bytes_transferred {direction=}
counter connects 2 2011-02-21T18:31:52Z
counter logins 2 2011-02-21T18:31:52Z
counter uploads 2 2011-02-21T18:31:55Z
counter session_time 1298310264 2011-02-21T18:31:55Z
counter commands {command=STAT} 4 2011-02-21T17:44:32Z
counter commands {command=PASV} 4 2011-02-21T18:31:55Z
counter commands {command=QUIT} 3 2011-02-21T18:31:55Z
counter commands {command=USER} 2 2011-02-21T18:31:52Z
counter commands {command=PASS} 2 2011-02-21T18:31:52Z
counter commands {command=TYPE} 2 2011-02-21T18:31:54Z
counter commands {command=STOR} 2 2011-02-21T18:31:55Z
counter responses {response=230} 3 2011-02-21T18:31:53Z
counter responses {response=211} 8 2011-02-21T17:44:32Z
counter responses {response=300} 4 2011-02-21T17:44:32Z
counter responses {response=227} 4 2011-02-21T18:31:55Z
counter responses {response=221} 3 2011-02-21T18:31:55Z
counter responses {response=220} 2 2011-02-21T18:31:52Z
counter responses {response=331} 2 2011-02-21T18:31:52Z
counter responses {response=200} 2 2011-02-21T18:31:54Z
counter responses {response=150} 2 2011-02-21T18:31:55Z
counter responses {response=226} 2 2011-02-21T18:31:55Z
